packages/components/text/src/text.tsx

```diff
@@ -219,14 +220,27 @@ const Detail = (props: TDetailProps) => {
   const theme = useTheme();
   warnIfMissingTitle(props, 'TextDetail');
   warnIfMissingContent(props, 'TextDetail');
+  if (props.as) {
+    const TextDetailElement = props.as;
```

Comment:
Nit: do we need the if-else? Can't we simply do `const TextDetailElement = props.as || div`? 🤔

Reply:
I like this.
I added support for `small` too, as another option to inline the text, besides `span` 😉